lib/msf/core/db_manager/host.rb

```diff
@@ -127,14 +127,6 @@ def get_host(opts)
   }
   end
 
-  # Look for an address across all comms
-  def has_host?(wspace,addr)
-  ::ApplicationRecord.connection_pool.with_connection {
-    address, scope = addr.split('%', 2)
-    wspace.hosts.find_by_address(addr)
-  }
-  end
```

Comment:
We talked about this on module hacking, and it seems like we'll want to keep this logic inside the the db_manager. Leaking concerns about databases into the layers of the metasploit console seems like something we'd want to avoid 👀 

Is it possible to keep this function here and wire it up properly? 🕵️